<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/images/icons_33.png</normaloff>:/icons/icons/images/icons_33.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadHDF5File"/>
   <addaction name="actionCloseHDF5File"/>
   <addaction name="separator"/>
   <addaction name="actionSliceXY"/>
   <addaction name="actionSliceXZ"/>
   <addaction name="actionSliceYZ"/>
   <addaction name="separator"/>
   <addaction name="actionVolumeRendering"/>
   <addaction name="action3DXY"/>
   <addaction name="action3DXZ"/>
   <addaction name="action3DYZ"/>
   <addaction name="separator"/>
   <addaction name="actionViewFrame"/>
   <addaction name="separator"/>
   <addaction name="actionFillSpace"/>
   <addaction name="separator"/>
   <addaction name="actionOrthogonal"/>
   <addaction name="separator"/>
   <addaction name="actionAlignToXY"/>
   <addaction name="actionAlignToXZ"/>
   <addaction name="actionAlignToYZ"/>
   <addaction name="actionAlignToXYFromBack"/>
   <addaction name="actionAlignToXZFromBack"/>
   <addaction name="actionAlignToYZFromBack"/>
   <addaction name="separator"/>
   <addaction name="actionExportImageFrom3DScene"/>
   <addaction name="separator"/>
   <addaction name="actionDatasets"/>
   <addaction name="actionSelectedDataset"/>
   <addaction name="actionInfo"/>
   <addaction name="action3DSettings"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadHDF5File"/>
    <addaction name="actionCloseHDF5File"/>
    <addaction name="actionExit"/>
    <addaction name="actionExportImageFrom3DScene"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuControls">
     <property name="title">
      <string>Controls</string>
     </property>
     <addaction name="actionInfo"/>
     <addaction name="actionDatasets"/>
     <addaction name="actionSelectedDataset"/>
     <addaction name="action3DView"/>
    </widget>
    <addaction name="actionSliceXY"/>
    <addaction name="actionSliceXZ"/>
    <addaction name="actionSliceYZ"/>
    <addaction name="separator"/>
    <addaction name="actionVolumeRendering"/>
    <addaction name="action3DXY"/>
    <addaction name="action3DXZ"/>
    <addaction name="action3DYZ"/>
    <addaction name="separator"/>
    <addaction name="actionViewFrame"/>
    <addaction name="separator"/>
    <addaction name="menuControls"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="SliceDockWidget" name="dockWidgetSliceXY">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
  </widget>
  <widget class="SliceDockWidget" name="dockWidgetSliceXZ">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
  </widget>
  <widget class="SliceDockWidget" name="dockWidgetSliceYZ">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
  </widget>
  <widget class="Window3DDockWidget" name="dockWidgetWindow3D">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
  </widget>
  <widget class="DatasetsDockWidget" name="dockWidgetDatasets">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
  </widget>
  <widget class="SelectedDatasetDockWidget" name="dockWidgetSelectedDataset">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
  </widget>
  <widget class="FileInfoDockWidget" name="dockWidgetInfo">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
  </widget>
  <widget class="Settings3DDockWidget" name="dockWidgetSettings3D">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
  </widget>
  <action name="actionLoadHDF5File">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_01.png</normaloff>:/icons/icons/images/icons_01.png</iconset>
   </property>
   <property name="text">
    <string>Open HDF5 simulation file</string>
   </property>
   <property name="toolTip">
    <string>Open HDF5 simulation file</string>
   </property>
  </action>
  <action name="actionCloseHDF5File">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_07.png</normaloff>:/icons/icons/images/icons_07.png</iconset>
   </property>
   <property name="text">
    <string>Close HDF5 file</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_08.png</normaloff>:/icons/icons/images/icons_08.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_09.png</normaloff>:/icons/icons/images/icons_09.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSliceXY">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_04.png</normaloff>:/icons/icons/images/icons_04.png</iconset>
   </property>
   <property name="text">
    <string>XY slice</string>
   </property>
   <property name="toolTip">
    <string>View XY slice</string>
   </property>
  </action>
  <action name="actionSliceXZ">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_05.png</normaloff>:/icons/icons/images/icons_05.png</iconset>
   </property>
   <property name="text">
    <string>XZ slice</string>
   </property>
   <property name="toolTip">
    <string>View XZ slice</string>
   </property>
  </action>
  <action name="actionSliceYZ">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_06.png</normaloff>:/icons/icons/images/icons_06.png</iconset>
   </property>
   <property name="text">
    <string>YZ slice</string>
   </property>
   <property name="toolTip">
    <string>View YZ slice</string>
   </property>
  </action>
  <action name="action3DXY">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_13.png</normaloff>:/icons/icons/images/icons_13.png</iconset>
   </property>
   <property name="text">
    <string>3D XY slice</string>
   </property>
  </action>
  <action name="action3DXZ">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_15.png</normaloff>:/icons/icons/images/icons_15.png</iconset>
   </property>
   <property name="text">
    <string>3D XZ slice</string>
   </property>
  </action>
  <action name="action3DYZ">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_14.png</normaloff>:/icons/icons/images/icons_14.png</iconset>
   </property>
   <property name="text">
    <string>3D YZ slice</string>
   </property>
   <property name="toolTip">
    <string>3D YZ slice</string>
   </property>
  </action>
  <action name="actionExportImageFrom3DScene">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_10.png</normaloff>:/icons/icons/images/icons_10.png</iconset>
   </property>
   <property name="text">
    <string>Export image from 3D scene</string>
   </property>
  </action>
  <action name="actionVolumeRendering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_32.png</normaloff>:/icons/icons/images/icons_32.png</iconset>
   </property>
   <property name="text">
    <string>Volume rendering</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>HDF5 Info</string>
   </property>
  </action>
  <action name="actionDatasets">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Datasets to view</string>
   </property>
  </action>
  <action name="actionSelectedDataset">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Selected dataset</string>
   </property>
  </action>
  <action name="actionAlignToXY">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_26.png</normaloff>:/icons/icons/images/icons_26.png</iconset>
   </property>
   <property name="text">
    <string>Align to XY</string>
   </property>
   <property name="toolTip">
    <string>Align to XY</string>
   </property>
  </action>
  <action name="actionAlignToXZ">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_27.png</normaloff>:/icons/icons/images/icons_27.png</iconset>
   </property>
   <property name="text">
    <string>Align to XZ</string>
   </property>
   <property name="toolTip">
    <string>Align to XZ</string>
   </property>
  </action>
  <action name="actionAlignToYZ">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_28.png</normaloff>:/icons/icons/images/icons_28.png</iconset>
   </property>
   <property name="text">
    <string>Align to YZ</string>
   </property>
   <property name="toolTip">
    <string>Align to YZ</string>
   </property>
  </action>
  <action name="actionAlignToXYFromBack">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_29.png</normaloff>:/icons/icons/images/icons_29.png</iconset>
   </property>
   <property name="text">
    <string>Align to XY from back</string>
   </property>
   <property name="toolTip">
    <string>Align to XY from back</string>
   </property>
  </action>
  <action name="actionAlignToXZFromBack">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_30.png</normaloff>:/icons/icons/images/icons_30.png</iconset>
   </property>
   <property name="text">
    <string>Align to XZ from back</string>
   </property>
   <property name="toolTip">
    <string>Align to XZ from back</string>
   </property>
  </action>
  <action name="actionAlignToYZFromBack">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_31.png</normaloff>:/icons/icons/images/icons_31.png</iconset>
   </property>
   <property name="text">
    <string>Align to YZ from back</string>
   </property>
   <property name="toolTip">
    <string>Align to YZ from back</string>
   </property>
  </action>
  <action name="actionViewFrame">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_34.png</normaloff>:/icons/icons/images/icons_34.png</iconset>
   </property>
   <property name="text">
    <string>View frame</string>
   </property>
   <property name="toolTip">
    <string>View frame</string>
   </property>
  </action>
  <action name="actionFillSpace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/images/icons_24.png</normaloff>:/icons/icons/images/icons_24.png</iconset>
   </property>
   <property name="text">
    <string>Fill 3D space</string>
   </property>
   <property name="toolTip">
    <string>Fill 3D space</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About k-Wave visualiser</string>
   </property>
  </action>
  <action name="actionOrthogonal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Isometric</string>
   </property>
   <property name="toolTip">
    <string>Toggle orthogonal/perspective</string>
   </property>
  </action>
  <action name="action3DView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3D view</string>
   </property>
  </action>
  <action name="action3DSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3D Settings</string>
   </property>
   <property name="toolTip">
    <string>3D Settings</string>
   </property>
  </action>
  <zorder>dockWidgetSettings3D</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Window3DDockWidget</class>
   <extends>QDockWidget</extends>
   <header location="global">window3ddockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DatasetsDockWidget</class>
   <extends>QDockWidget</extends>
   <header location="global">datasetsdockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SelectedDatasetDockWidget</class>
   <extends>QDockWidget</extends>
   <header location="global">selecteddatasetdockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SliceDockWidget</class>
   <extends>QDockWidget</extends>
   <header location="global">slicedockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileInfoDockWidget</class>
   <extends>QDockWidget</extends>
   <header location="global">fileinfodockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Settings3DDockWidget</class>
   <extends>QDockWidget</extends>
   <header location="global">settings3ddockwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInfo</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockWidgetInfo</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>1201</x>
     <y>579</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelectedDataset</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockWidgetSelectedDataset</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>953</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSliceXY</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockWidgetSliceXY</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSliceYZ</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockWidgetSliceYZ</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSliceXZ</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockWidgetSliceXZ</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action3DView</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockWidgetWindow3D</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>574</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDatasets</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockWidgetDatasets</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>901</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidgetSliceXY</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionSliceXY</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidgetSliceXZ</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionSliceXZ</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidgetSliceYZ</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionSliceYZ</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidgetDatasets</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionDatasets</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>901</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidgetSelectedDataset</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionSelectedDataset</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>901</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidgetInfo</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionInfo</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>901</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidgetWindow3D</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>action3DView</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action3DSettings</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockWidgetSettings3D</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>900</x>
     <y>646</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setImageXYFromData()</slot>
  <slot>setImageXZFromData()</slot>
  <slot>setImageYZFromData()</slot>
 </slots>
</ui>
